### PR TITLE
Temporarily remove weekly test list

### PR DESCRIPTION
**Pull request by smlambert**
Many jobs in queue during release, we will have to assess what is possible during the surge of capacity requirements during a release.

### CODE REVIEW

**Review by gdams — APPROVED** [review]

**Review by sxa — APPROVED** [review]

**Comment by smlambert** [comment]
I can wait until the checks are completed to merge, will want to merge prior to respins during release to avoid waiting on machines to run this additional set of tests.

FYI @lumpfish

**Review by karianna — APPROVED** [review]
Approved

**Comment by karianna** [comment]
As discussed in the release channel we are rolling back the new weekly test suites.

**Pull request merged**
